docs/installation.rst

```diff
@@ -12,9 +12,17 @@ To install conda-devenv, run this command in your terminal:
 
 .. code-block:: console
 
-    $ pip install conda_devenv
+    $ conda install conda-devenv -c conda-forge
 
-This is the preferred method to install conda-devenv, as it will always install the most recent stable release. 
+
+Or if you prefer PIP:
```

Comment:
Does this work?

If we are not sure we should probably just remove this option from the help.